docs/content/getting-started/_index.md

```diff
@@ -92,17 +125,155 @@ class App extends Component {
+
+Theme variables will impact all components on the page where this CSS is loaded.
+
+
+```
+// your-page.css
+:root {
+    --pfe-theme--color--ui-accent: green;
+    --pfe-theme--color--surface--darker: navy;
+}
+```
+
+
+_Note_: overriding local variables (i.e., --pfe-cta--foo) will not work at this level.  Those overrides can only be done on the component via tag name, class, or ID.
+
+
+#### Page-level CSS, component variables
+
+
+```
+/* your-page.css */
+pfe-cta {
+    --pfe-cta--BackgroundColor: green;
+}
+pfe-band {
+    --pfe-band--Padding--vertical: 34px;
+}
+```
+
+
+
+### Component-level inline CSS
+
+As a last resort, you may choose to override variables with inline styles. This could be desirable if you only need one component to deviate from the design system. Note that this incurs some level of risk, especially related to colors, as you are opting out of the color combinations in the system.
+
+
+```
+<pfe-cta style="--pfe-cta--BackgroundColor:pink">
+```
+
+
+## 6. Victory!
+
+Be sure to browser test within your own site or application. If you run into any trouble, please file an issue in the [issue queue](https://github.com/patternfly/patternfly-elements/issues).
+
+<hr/>
+
+
+## FAQ
+
+### Should I use on=dark or color=darkest on my container? What's the difference?
+
+*   `on=dark` is being deprecated. Instead, custom classes already on the page should set broadcast values.
+*   The original goal for `on=dark` was all about context. But instead of having to provide another attribute somewhere, the card or a band or another container will inform any nested components that the text color needs to change through the set of broadcast variables. Existing broadcast vars (defined in `$BROADCAST-VARS`)  include: `text`, `ui-link`, `ui-link--hover`, `ui-link--visited`, `ui-link--focus`.
+*   If your page has a custom background color for a band or card, you should set the background and also set the broadcast colors.
+
+
+```
+.custom-dark-band {
+  background: black;
+  --pfe-broadcasted--color--text:             var(--pfe-theme--color--text--on-dark, #fff);
+  --pfe-broadcasted--color--ui-link:          var(--pfe-theme--color--ui-link--on-dark, #73bcf7);
+  --pfe-broadcasted--color--ui-link--hover:   var(--pfe-theme--color--ui-link--on-dark--hover, pink);
+  --pfe-broadcasted--color--ui-link--visited: var(--pfe-theme--color--ui-link--on-dark--visited, pink);
+  --pfe-broadcasted--color--ui-link--focus:   var(--pfe-theme--color--ui-link--on-dark--focus, pink);
+}
+```
+
+In themes (like the advanced-theme.css file from the Red Hat Theme ) we apply broadcast variables to plain links, because they are light DOM and also have default colors applied by the browser. This CSS file not only includes variables but also styles for headlines and links on the page. It sets the colors for these elements using a CSS variable, which web components can change the value of.
+
+For example, advanced-theme.css includes
```

Comment:
I think there should be a colon at the end of this sentence
For example, advanced-theme.css includes:

Reply:
updated, thx!